plugins/module_utils/network/eos/facts/lacp_interfaces/lacp_interfaces.py

```diff
@@ -99,5 +99,7 @@ def render_config(self, spec, conf):
         config["name"] = utils.parse_conf_arg(conf, "interface")
         config["port_priority"] = utils.parse_conf_arg(conf, "port-priority")
         config["rate"] = utils.parse_conf_arg(conf, "rate")
+        if not config.get("timer"):
+            config["timer"] = utils.parse_conf_arg(conf, "timer")
```

Comment:
Is this right? I would think it would just be this now
```suggestion
        config["timer"] = utils.parse_conf_arg(conf, "timer")
```